pkg/gui/presentation/files.go

```diff
@@ -21,13 +22,14 @@ const (
 func RenderFileTree(
 	tree filetree.IFileTree,
 	submoduleConfigs []*models.SubmoduleConfig,
-	showFileIcons bool,
+	showFileIcons,
```

Comment:
Style nitpick: I don't very much like go's syntax to declare multiple parameters of the same type in one go (in general), but when they are on separate lines, I definitely think they should each repeat the type.